src/things/comment.rs

```diff
@@ -114,10 +114,12 @@ impl Shred for Comment {
         );
 
         headers.insert("User-Agent", config.user_agent.parse().unwrap());
+        
+        let new_comment_text = if config.replacement_comment.is_empty() {LOREM_IPSUM.to_string()} else {config.replacement_comment.to_string()};
```

Comment:
I'd set the CLI default to be `LOREM IPSUM`. That way you don't need to do a check here at all, and the CLI help menu will display it.